presto-hive/src/main/java/com/facebook/presto/hive/metastore/ThriftHiveMetastore.java

```diff
@@ -769,6 +737,38 @@ public void revokeTablePrivileges(String databaseName, String tableName, String
         }
     }
 
+    private PrivilegeBag buildPrivilegeBag(HiveMetastoreClient metastoreClient,
+                                           String databaseName,
+                                           String tableName,
+                                           String grantee,
+                                           Set<PrivilegeGrantInfo> privilegeGrantInfos)
+            throws TException
+    {
+        final String PUBLIC_ROLE_NAME = "public";
```

Comment:
this is causing the build to fail since is it is not an allowed variable name.  This should be a class level constant `public static final String PUBLIC_ROLE_NAME = "PUBLIC";` declared at the top of the class